scripts.js

```diff
@@ -0,0 +1,14 @@
+function add_doggo() {
+    var src = "https://dog.ceo/api/breed/shiba/images";
+    show_image("https://dog.ceo/api/breed/shiba/images", 500,500, "Doggo");
+}
+
+
+function show_image(src, width, height, alt) {
+    var img = document.createElement("img");
```

Comment:
Looks good!  You could also have the image just in the HTML file since you know there will always be an image.

Reply:
So I can take https://dog.ceo/api/breed//images/random and put that in a 
    var img = document.createElement("img");?